src/styles/global.css

```
@@ -209,6 +209,188 @@ body[data-mobile-nav-open] {
+.post-body strong {
+	color: var(--text-main);
+	font-weight: 700;
+}
+
+.post-body a {
+	color: var(--accent);
+	text-decoration-line: underline;
+	text-decoration-thickness: 0.08em;
+	text-underline-offset: 0.18em;
+	text-decoration-color: currentColor;
+	word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)